packages/grafana-data/src/types/dataLink.ts

```diff
@@ -102,6 +102,11 @@ export interface LinkModel<T = any> {
   // When a click callback exists, this is passed the raw mouse|react event
   onClick?: (e: any, origin?: any) => void;
   oneClick?: boolean;
+
+  interpolation?: {
```

Comment:
```suggestion
  interpolation?: {
```
I would consider other attribute names such as `source` or `params`.

Reply:
what about `interpolatedParams`?